rasa/shared/core/domain.py

```diff
@@ -327,7 +327,21 @@ def _transform_intent_properties_for_internal_use(
         """
         name, properties = list(intent.items())[0]
 
-        properties.setdefault(USE_ENTITIES_KEY, True)
+        if properties:
+            properties.setdefault(USE_ENTITIES_KEY, True)
+        else:
+            raise InvalidDomain(
+                f"In the `domain.yml` file, the intent `{name}` cannot have value of"
+                f" `{type(properties)}`. This can happen if you add an intent with the"
+                f" `:` (colon character) after it and do not add any"
+                f" further information to this specific intent."
+                f" If you remove the `:` character it should work. Please see "
```

Comment:
I would add more certainty to this phrase. Something like `You need to remove the ":" character to make it work.`